conference/react-js/src/reducers/forgot-password.js

```diff
@@ -0,0 +1,17 @@
+import actions from '../constants/actions-types';
+
+const { SHOW_SUCCESS_RESET_PASSWORD_MESSAGE,
+  HIDE_SUCCESS_RESET_PASSWORD_MESSAGE } = actions;
+
+function forgotPassword(isForgotPasswordSent, action) {
+  const { type } = action;
+  let res = false;
+  if (type === SHOW_SUCCESS_RESET_PASSWORD_MESSAGE) {
```

Comment:
And here
 the same situation like in ```forgot-password-error-text.jsx```.
You can return value directly from "if" statement.